sources/controllers/Register.controller.php

```diff
@@ -1258,4 +1265,42 @@ public function action_registerCheckUsername()
 
 		$context['valid_username'] = !$errors->hasErrors();
 	}
+
+	public function action_agrelang()
```

Comment:
Why not `action_agreelang`? Does that already exist? Something against double vowels? :-P